File: java/questao3.java
import java.util.Scanner;
public class questao3 {
    public static void main(String args[]){

   System.out.print("Digite o valor da temperatura em Celsius: ");
   Scanner scan = new Scanner(System.in);
   float temp = scan.nextFloat();
   System.out.print("O valor da temperatura convertida é: " +(temp*1.8+32)+"F");
    }

}

File: questao3.c
#include <stdio.h>
#include <stdlib.h>

int main(){
   float temp;

   printf("Digite o valor da temperatura em celsius: ");
   scanf("%f", &temp);
   printf("%.1f em Celsius sao %.1f em Farenheit", temp, temp*1.8+32);

}